src/itemloader.h

```diff
@@ -103,6 +104,10 @@ enum clientVersion_t {
 	CLIENT_VERSION_1035 = 55,
 	CLIENT_VERSION_1076 = 56,
 	CLIENT_VERSION_1098 = 57,
+	CLIENT_VERSION_10100 = 58,
```

Comment:
Every other version should have an extra 0 too then

Reply:
There is actually no version 10.100, as the last outdated client is version 11.00

Reply:
No. That's 10.100, not 10.10. And 10098 will look even weirder.

Reply:
![image](https://user-images.githubusercontent.com/26801045/140417971-8038fe64-4c5d-4f21-9f4f-cc5b8836e894.png)

![image](https://user-images.githubusercontent.com/26801045/140417847-be5df7de-9233-452d-a831-74b26b739c7b.png)
Just test with the outdated client

Reply:
Actually there is no version 10.100, the cip kept it only for aesthetics on the client.
If you add a print and login with the legacy client you will see that it is actually version 11.00 and not 10.000

![image](https://user-images.githubusercontent.com/26801045/140419368-ffed2876-9dbd-4e9f-920b-7dc0625614e8.png)
![image](https://user-images.githubusercontent.com/26801045/140419290-861af950-acd7-4fa7-9675-87f94d222d07.png)
![image](https://user-images.githubusercontent.com/26801045/140419317-5ab19ee7-11c5-441d-839c-574372d15877.png)

Reply:
updated